main-extract.ts

```diff
@@ -300,7 +302,8 @@ const generatePreviewClip = (
       concat += '[' + (current - 1) + ':V]' + '[' + (current - 1) + ':a]';
       current++;
     }
-    concat += 'concat=n=' + (totalCount - 1) + ':v=1:a=1[v][a];[v]scale=-2:' + screenshotHeight + '[v2]';
+    concat += 'concat=n=' + (totalCount - 1) + ':v=1:a=1[v][a];[v]scale=-2:' +
+              (clipHalfRez ? screenshotHeight / 2 : screenshotHeight) + '[v2]';
```

Comment:
I assume ffmpeg will handle it, but have to tried halving an _odd_ number?

Reply:
Currently I've hardcoded these heights:
```
'144', '216', '288', '360', '432'
```
No other ones are allowed. Halving any of them seems to work fine when I did spot-testing in the app 😅